.yamllint_defaults.yml

```diff
@@ -10,5 +10,7 @@ rules:
   brackets:
     max-spaces-inside: 1
     level: error
-  truthy: disable
+  truthy:
+    allowed-values: ["yes", "no", "true", "false"]
```

Comment:
IMHO this huge commit buries this important policy change. I remember that there was an agreement to avoid such huge commits. Also, I still prefer to have either yes/no or true/false as allowed values instead of both pairs.

Reply:
> IMHO this huge commit buries this important policy change. I remember that there was an agreement to avoid such huge commits. Also, I still prefer to have either yes/no or true/false as allowed values instead of both pairs.

ok - well - I'm not sure how to cherry pick changes from the template when syncing changes to all of the system roles repos - suggestions welcome

Reply:
> I remember that there was an agreement to avoid such huge commits. 

Then please, please review the template changes, and let's please get the template changes into a form where *all* of the repo maintainers can agree that, once the change is committed to template, that same exact change can be synced to all of the repos - otherwise, this process just won't scale

Reply:
I added my comments about the values to https://github.com/linux-system-roles/template/pull/34#issuecomment-645402359.

Still, this is a squashed commit from the template instead of the individual commits originally used committed to the template and the commit message here does not even state that it will add/change anything related to yamllint/truthy.

> > IMHO this huge commit buries this important policy change. I remember that there was an agreement to avoid such huge commits. Also, I still prefer to have either yes/no or true/false as allowed values instead of both pairs.
> 
> ok - well - I'm not sure how to cherry pick changes from the template when syncing changes to all of the system roles repos - suggestions welcome

You can add the template repo as an additional remote to the working copies of the roles, then fetch and cherry pick commits from there. I did it the other way around when merging commits back from the network role to the template repo.

A probably cleaner solution would be to move all the template tools/code into its own python package that could be installed inside tox and do proper releases with release notes, there, then once something is done it can be released on pypi and would be picked up by the roles. Then the PRs in the roles would only contain the specific changes for the roles.

Reply:
fixed in https://github.com/linux-system-roles/network/pull/261/commits/e2c5cd73388c6d75f3376f44aa5f2c799199c335 - added
```
rules:
    truthy: disable
```
to .yamllint.yml until such time as we can agree on what boolean format to enforce in system roles

Reply:
> You can add the template repo as an additional remote to the working copies of the roles, then fetch and cherry pick commits from there.

Which is completely different than the current method of using sync-template.sh . . . which would require a significant rewrite of that script to implement such a procedure, or a completely new script.

Given this, and the other comments, I would prefer to move to a library based approach, possibly using a custom tox plugin as we have discussed in the past - something like https://github.com/greg-hellings/tox-ansible which https://github.com/oasis-roles uses - that would also increase our collaboration with the oasis group which we have been wanting to do

Reply:
So what I would like to do is to merge this PR as-is (or with some other minor modifications based on further review comments), then at some point in the near future implement a plugin based CI model.